test/Microsoft.AspNetCore.Razor.Tools.Test/ServerCommandTest.cs

```diff
@@ -47,10 +47,10 @@ public void WritePidFile_WorksAsExpected()
             }
             finally
             {
-                // Delete the file in case the test fails.
-                if (File.Exists(path))
+                // Cleanup after the test.
+                if (Directory.Exists(directoryPath))
                 {
-                    File.Delete(path);
+                    Directory.Delete(directoryPath);
```

Comment:
Might have to say `recursive: true`. I don't think `Directory.Delete` deletes files by default